app/controllers/rentals_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'will_paginate/array'
```

Comment:
I think I was debugging and thought that was necessary, it works perfectly fine when i remove it.